cmd/glasskube/cmd/update.go

```diff
@@ -47,6 +60,7 @@ var updateCmd = &cobra.Command{
 				fmt.Fprintf(os.Stderr, "❌ update failed: %v\n", err)
 				os.Exit(1)
 			}
+			printTransaction(*tx)
```

Comment:
`printTransaction(*tx)` prints the transaction to the terminal, so that the user can decide whether they want to apply these updates or not. Therefore it should be called before the user is asked to confirm (`YesNoPrompt`).